Control/DeepSeq.hs

```diff
@@ -179,6 +179,13 @@ instance GNFData arity a => GNFData arity (M1 i c a) where
   grnf args = grnf args . unM1
   {-# INLINEABLE grnf #-}
 
+#if __GLASGOW_HASKELL__ >= 800
+  -- @since 1.4.5.0
```

Comment:
This isn't a proper haddock doc-string;  I take it that `URec` didn't exist prior to base-4.9; it would be useful to state this type and instance are only available starting with base-4.9

Reply:
Good catch. I've submitted #48 with a fix—is this to your liking?